scripts/test

```diff
@@ -25,7 +25,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
             *.py ./**/*.py \
             *.md \
             docs/*.rst docs/**/*.rst \
-            docs/*.ipynb docs/**/*.ipynb
+            # docs/*.ipynb docs/**/*.ipynb
```

Comment:
Instead of this, adding "ressources" to `.codespellignore` seems to work.
If we're including the bilingual name here, should it go in the constants file too?

Reply:
"Ressources" is part of the constants file that is reproduced as a cell output in the notebook to provide an example of a STAC Collection. I've left the cell outputs in the notebook so a user can browse expected results without running the notebook in an environment/mybinder/container. Some of these outputs may from the dataset source, which is outside of our control, so I'm unsure if it makes to include cell outputs **and** spellcheck the notebook.

Reply:
Why isn't it failing against `constants.py`?

Reply:
Apparently it only looks for common typos, so most words it doesn't know won't set it off.

Reply:
If we're going to remove this, just delete the line. Don't comment it.